itou/www/apply/forms.py

```diff
@@ -453,8 +453,8 @@ class AnswerForm(forms.Form):
 
 class AcceptForm(JobAppellationAndLocationMixin, forms.ModelForm):
     """
-    Allow an SIAE to accept a job application.
-    If SIAE is a GEIQ, add specific fields (contract type, number of hours per week)
+    Allow an company to accept a job application.
```

Comment:
```suggestion
    Allow a company to accept a job application.
```
Cela vaut probablement le coup de greper tous les `"an company"` qui pourrait rester :sweat_smile:

Reply:
Je vais faire un gros commit dans le lot 10 plutôt :)